extensions/adobe/experience/content-telemetry/webelement.schema.json

```diff
@@ -0,0 +1,118 @@
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/content-telemetry/webelement",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Telemetry Web Element",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Web element metadata",
+  "definitions": {
+    "webelement": {
+      "properties": {
+        "repo:id": {
+          "title": "Element Id",
+          "type": "string",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "description": "The unique ID of the web element."
+        },
+        "repo:lastModifiedDate": {
+          "type": "string",
+          "format": "date-time",
+          "meta:usereditable": false,
+          "examples": ["2004-10-23T12:00:00-06:00"],
+          "description": "The server date and time when the resource was last modified in the repository, such as when a new version of an asset is uploaded or a directory's child resource is added or removed. The date time property should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."
+        },
+        "xdm:elementType": {
+          "title": "Element Type",
+          "type": "string",
+          "description": "The type of the element's component.",
+          "meta:usereditable": false
+        },
+        "dc:title": {
+          "title": "Title",
+          "type": "string",
+          "description": "The title of the web element"
+        },
+        "xdm:language": {
+          "type": "string",
+          "pattern": "^(((([A-Za-z]{2,3}(-([A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-([A-Za-z]{4}))?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-([0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(x(-[A-Za-z0-9]{1,8})+))?)|(x(-[A-Za-z0-9]{1,8})+)|((en-GB-oed|i-ami|i-bnn|i-default|i-enochian|i-hak|i-klingon|i-lux|i-mingo|i-navajo|i-pwn|i-tao|i-tay|i-tsu|sgn-BE-FR|sgn-BE-NL|sgn-CH-DE)|(art-lojban|cel-gaulish|no-bok|no-nyn|zh-guoyu|zh-hakka|zh-min|zh-min-nan|zh-xiang)))$",
+          "description": "Specifies the language of this web element. the language property should conform to BPC 47, for example `en-GB`."
+        },
+        "xdm:asset": {
+          "title": "Asset",
+          "description": "The asset referred by the web element.",
+          "$ref": "https://ns.adobe.com/experience/content-telemetry/webasset"
+        },
+        "xdm:linkUrl": {
+          "title": "LinkUrl",
+          "type": "string",
+          "format": "uri",
+          "description": "The URL the element links to."
+        },
+        "xdm:text": {
+          "title": "Text",
+          "type": "string",
+          "description": "The element's text."
+        },
+        "xmp:keywords": {
```

Comment:
what you describe below isn't consistent with xmp:keywords.  You can't use the same but redefine what it means...

Reply:
See ISO 16684-3 on how to map XMP's RDF+XML to JSON-LD.

Reply:
Or use another name if you want this specific set of properties.  (and if you do, then the children are `xmp:value`, `dc:language` and `xdm:importance`

Reply:
`xmp:Keywords` is defined here https://git.corp.adobe.com/caf/xcm/blob/master/schemas/assets/asset.schema.json#L310 so you simply need to include that in your list of contexts and it will pull in all of the asset schema members.